vendor/parser-extended/parser.mly

```diff
@@ -1828,8 +1828,8 @@ class_expr:
       { $1 }
   | FUN attributes class_fun_def
       { wrap_class_attrs ~loc:$sloc $3 $2 }
-  | let_bindings(no_ext) IN class_expr
-      { class_of_let_bindings ~loc:$sloc $1 $3 }
+  | let_bindings(no_ext) mkrhs(IN) class_expr
+      { class_of_let_bindings ~loc:$sloc ~loc_in:$2.loc $1 $3 }
```

Comment:
I find it weird to use `mkrhs` but then discard the result. Otherwise:
```suggestion
  | let_bindings(no_ext) IN class_expr
      { class_of_let_bindings ~loc:$sloc ~loc_in:(make_loc $loc($2)) $1 $3 }
```

What do you think ?

Reply:
Yes thats fine